examples/platforms/efr32/alarm.c

```diff
@@ -142,11 +165,18 @@ void efr32AlarmProcess(otInstance *aInstance)
             otPlatAlarmMilliFired(aInstance);
         }
     }
-
+    else
+    {
+        if (RAIL_IsTimerExpired(gRailHandle))
+        {
+            expires = sAlarmT0 + sAlarmDt;
+            RAIL_SetTimer(gRailHandle, expires, RAIL_TIME_ABSOLUTE, RAILCb_TimerExpired);
+        }
+    }
 exit:
     return;
 }
 
-void RAILCb_TimerExpired(void)
+static void RAILCb_TimerExpired(RAIL_Handle_t aHandle)
```

Comment:
nit: If this is moved to the top of the file, could we get rid of the forward declaration?